Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -43,6 +43,8 @@
     'Outgoing': 'Out',
     'Incoming And Outgoing': 'Both'
 }
+OUTGOING_MIRRORED_FIELDS = ['comment', 'status', 'owner', 'urgency']
```

Comment:
In our outgoing mapper are we mapping status also to 'status_label'?

Reply:
nope, we don't use it that way inn the new outgoing mapper

Reply:
So we are telling users we are able to mirror in the `status_label` but not out?